components/eam/src/physics/crm/ndrop.F90

```diff
@@ -1074,20 +1073,23 @@ subroutine dropmixnuc( &
             end do
          end do
 
-if (SPCAM_mmf) then
-! turbulent mixing for gas species 
+
+
 #if (defined MODAL_AERO)
-   do m=1, pcnst
-      if(species_class(m).eq.spec_class_gas) then
-         flxconv = 0.0_r8
-         zerogas(:) = 0.0_r8
-         call explmix(rgascol(1,m,nnew),zerogas,ekkp,ekkm,overlapp,overlapm,  &
-                    rgascol(1,m,nsav),zero, flxconv, pver,dtmix,&
-                      .true., zerogas)
-      end if
-   end do
+         ! turbulent mixing for gas species 
+         if (use_MMF .and. present(do_mmf)) then
+            do m=1, pcnst
+               if(species_class(m).eq.spec_class_gas) then
+                  flxconv = 0.0_r8
+                  zerogas(:) = 0.0_r8
+                  call explmix(rgascol(1,m,nnew),zerogas,ekkp,ekkm,overlapp,overlapm,  &
+                             rgascol(1,m,nsav),zero, flxconv, pver,dtmix,&
```

Comment:
Some nitpicking, but so long as you're changing whitespace here, may as well use consistent formatting.

Reply:
Fixed.